docs/DeveloperGuide.adoc

```diff
@@ -49,7 +49,7 @@ This will generate all resources required by the application and tests.
 
 === Verifying the setup
 
-. Run the `seedu.address.MainApp` and try a few commands
+. Run the `seedu.planner.MainApp` and try a few commands
```

Comment:
nit: My PR will update `.adoc` too. Please revert this line to avoid merge conflict.